pretixscan/app/build.gradle

```diff
@@ -17,7 +17,7 @@ android {
     defaultConfig {
         applicationId "eu.pretix.pretixscan.droid"
         minSdkVersion 17
-        targetSdkVersion 30
+        targetSdkVersion 31
```

Comment:
Should be `33`?

Reply:
oops, yeah, i got confused with the many apps. 31 is the minum we need for google play, but we can do 33 straight away if it works. another round of testing then :D